ddtrace/debugging/_probe/remoteconfig.py

```diff
@@ -220,7 +211,7 @@ def _next_status_update_timestamp(self):
     def _dispatch_probe_events(self, prev_probes, next_probes):
         new_probes = [p for _, p in next_probes.items() if _ not in prev_probes]
         deleted_probes = [p for _, p in prev_probes.items() if _ not in next_probes]
-        modified_probes = [p for _, p in next_probes.items() if _ in prev_probes and probe_modified(p, prev_probes[_])]
+        modified_probes = []  # DEV: Probes are currently immutable
```

Comment:
not sure about this one. as now probe can be modified: condition changed or template changed.
Do we have a test that only change the probe condition or depth and to see it is working as expected?

Reply:
I'd keep that as out-of-scope for this PR since here we are cleaning up the `active` attribute, and with that gone the probe would effectively become immutable. We'll tackle mutability via conditions and limits in another PR